config/crd/bases/endpointmonitor.stakater.com_endpointmonitors.yaml

```diff
@@ -3,8 +3,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.9.0
-  creationTimestamp: null
+    controller-gen.kubebuilder.io/version: v0.14.0
```

Comment:
What made you require a version bump here?

Reply:
The version bump is required because the `make install` function segfaults due to a bug in v0.9 in controller-gen